angular/polyfills-es5.js

```diff
@@ -1 +1 @@
\ No newline at end of file
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function has a complexity of 40. Maximum allowed is 20.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5486792)